modules/logging-create-clf.adoc

```diff
@@ -19,36 +16,32 @@ You need administrator permissions for the namespace where you create the `Clust
 .ClusterLogForwarder resource example
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
-metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+metadata:  
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
-  serviceAccountName: <service_account_name> <3>
+  managementState: Managed
+  outputs:
+  - name: <output_name> #<1>
+    type: <output_type> #<2>
   pipelines:
```

Comment:
pipelines also can contain `filterRefs`.  Not sure its necessary to show that here, but it is a very important part of the pipeline.

Reply:
added filterRefs